torch/distributed/distributed_c10d.py

```diff
@@ -1714,6 +1714,10 @@ def broadcast_object_list(object_list, src=0, group=None):
         src (int): Source rank from which to broadcast ``object_list``.
         group: (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used. Default is ``None``.
+        dist_device (``torch.device``, optional): If nont None, then intermediate
```

Comment:
nit: `nont` -> `not` and maybe change `None` to be \`\` `None` ``.

Reply:
great catch.